src/LoggerTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Component;
+
+use Monolog\Handler\TestHandler;
+use PHPUnit\Framework\TestCase;
+
+class LoggerTest extends TestCase
+{
+    public function testLoggingWithDefaultHandlers(): void
+    {
+        $errorHandler = new TestHandler();
+        $logHandler = new TestHandler();
+        $logger = new Logger([
+            $errorHandler,
+            $logHandler,
+        ]);
+
+        $logger->error('Test error');
+        $logger->warning('Test warning');
+        $logger->notice('Test notice');
+
+        $this->assertTrue($errorHandler->hasErrorRecords());
```

Comment:
tahle assertion neni potreba, nasledujici to pokryva - totez dole

Reply:
celý test je pryč